# If we don't comment this out, the daemon will fail to start
--- mqttwarn/examples/basic/mqttwarn.ini.orig	2019-12-27 00:39:16 UTC
+++ mqttwarn/examples/basic/mqttwarn.ini
@@ -50,7 +50,7 @@ loglevel  = DEBUG
 ; --------
 
 ; path to file containing self-defined functions for formatmap and datamap
-functions = 'samplefuncs.py'
+#functions = 'samplefuncs.py'
 
 ; name the service providers you will be using.
 launch    = file, log
